src/ServiceControl/HeartbeatMonitoring/LegacyHandler.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace ServiceControl.HeartbeatMonitoring
```

Comment:
Is this one only to consume messages that might still be in the queue when upgrading?